.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: node_js
 node_js:
-    - "0.10"
-    - "0.12"
     - "4.3"
     - "4"
     - "5"
```

Comment:
"node" tests the latest node version, which is currently v6. I added v6 explicitly in a follow-up, so that we keep testing that even when v7 is released.